src/controllers/expense.controller.js

```diff
@@ -0,0 +1,90 @@
+const {
+  getExpenses,
+  getExpenseById,
+  create,
+  remove,
+  update,
+} = require('../services/expenses.service');
+const userService = require('../services/user.service');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  res.statusCode = 200;
+  res.send(getExpenses(userId, categories, from, to));
+};
+
+const getExpense = (req, res) => {
```

Comment:
You use like expenseController.[your method] so we already know that it related to expense
```suggestion
const getById = (req, res) => {
```